src/components/Clock.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class Clock extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { date: new Date() };
```

Comment:
Можешь сразу вызвать метод toLocaleTimeString() для date, чтоб потом его не дописывать
```suggestion
    this.state = { date: new Date().toLocaleTimeString() };
```